modules/dynamic-plugins/ref-rh-supported-plugins.adoc

```diff
@@ -96,7 +96,7 @@
 |`./dynamic-plugins/dist/backstage-community-plugin-scaffolder-backend-module-quay-dynamic`
 
 
-|RBAC  |`https://npmjs.com/package/@backstage-community/plugin-rbac/v/1.33.5[@backstage-community/plugin-rbac]` |1.33.5 
+|RBAC  |`https://npmjs.com/package/@backstage-community/plugin-rbac/v/1.33.6[@backstage-community/plugin-rbac]` |1.33.6 
```

Comment:
This corresponds to a [plugin documentation change](https://github.com/backstage/community-plugins/blob/main/workspaces/rbac/plugins/rbac/CHANGELOG.md#1336) so it looks safe to pick up